### ISSUE TITLE
Remove snow's biome, but merge into into minetest_games 

### ISSUE THREAD

**Issue by HybridDog** [Splizard/minetest-mod-snow#37]
Snow's mapgen is disabled by default now.
As I wrote [there](https://forum.minetest.net/viewtopic.php?f=47&t=19841), the biome size settings of the snow mod do not work right. I think the snow biome should generate into the default snow biome.

Currently snow's biome shows a lot more variety than minetest's snow biome, e.g. snow adds snowdrifts, whereas in minetest's biome snow nodes all have the same height.
Putting snowdrifts into minetest's snow biome can be done, but putting the snow shrubs and similar nodes is more difficult because there are no dry shrubs there which can be replaced.

**Comment by Splizard** [comment]
I would agree with this, the current state of this mod's mapgen is a bit buggy. 
As for any future direction, I think that icy biomes and icebergs would be a nice addition to the mod. It's not something I will be looking into at the moment, however, my website is up and running now so I am happy to make future stable releases of snow mod and update the forum page when needed :+1:

**Comment by HybridDog** [comment]
Here are some of the features:
Snow drifts are added, the Weierstraß fractal function is used to calculate them.
![snowslope](https://user-images.githubusercontent.com/3192173/43780452-e86c2786-9a5a-11e8-9167-c7e1e0584419.jpg)
So now and then snowy slopes are put where players can walk around without need for jumping.

Smooth biome borders are available, the maximum height of snow (drifts) depends on the distance to the chunk border.
![biome_border](https://user-images.githubusercontent.com/3192173/43780572-30c9c308-9a5b-11e8-8a39-56d49bf57a6f.jpg)

Dried plants are added for decoration: dry shrub, snow shrub, dried flowers
![dried_plants](https://user-images.githubusercontent.com/3192173/43780651-5bf400ca-9a5b-11e8-87db-eed408029bd8.jpg)

Ice has lots of variations, I can't remember them myself:
![ice_transit](https://user-images.githubusercontent.com/3192173/43780727-8a561ffc-9a5b-11e8-8e4b-30ecd38b0bc1.jpg)
![ice_variat](https://user-images.githubusercontent.com/3192173/43780748-9ef8daf8-9a5b-11e8-83b3-c64aa91ef0cc.jpg)

The default apple tree is has its counterpart in the snow biome, the other trees are snow's trees which are available in default now.
![snow_trees](https://user-images.githubusercontent.com/3192173/43780834-cd87f250-9a5b-11e8-8e0b-a0a7351dc41b.jpg)
Snow puts snow onto these pine trees if and only if they are in the snow biome.

In a specific sub-biome, default apple tree is replaced with this stone pillar: 
![treestone](https://user-images.githubusercontent.com/3192173/43780948-1b0fc868-9a5c-11e8-81bc-a9f5c516b151.jpg)
Papyrus and other nodes are also handled this way.

There are kind of sub-biomes:
https://github.com/Splizard/minetest-mod-snow/blob/036f50296861d020e0c5faaae3951483b48d3042/src/mapgen_v6.lua#L575-L578
